cypress/plugins/index.js

```diff
@@ -11,10 +11,19 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
+
 const cucumber = require('cypress-cucumber-preprocessor').default
-module.exports = on => {
+const dotenv = require('dotenv')
+
+module.exports = (on, config) => {
   // (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  const { parsed } = dotenv.config({ path: require.resolve('../../backend/.env') })
+  config.env.NEO4J_URI = parsed.NEO4J_URI
+  config.env.NEO4J_USERNAME = parsed.NEO4J_USERNAME
+  config.env.NEO4J_PASSWORD = parsed.NEO4J_PASSWORD
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-20T08:32:37Z" title="Monday, January 20th 2020, 9:32:37 am +01:00">Jan 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -11,10 +11,19 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
+
 const cucumber = require('cypress-cucumber-preprocessor').default
-module.exports = on => {
+const dotenv = require('dotenv')
+
+module.exports = (on, config) => {
   // (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  const { parsed } = dotenv.config({ path: require.resolve('../../backend/.env') })
+  config.env.NEO4J_URI = parsed.NEO4J_URI
+  config.env.NEO4J_USERNAME = parsed.NEO4J_USERNAME
+  config.env.NEO4J_PASSWORD = parsed.NEO4J_PASSWORD
```


I really like this solution... I saw how to do this, but didn't think to use it here. Nice catch!